src/components/layouts/pages/inspect/Inspect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getPdfFiles } from '../../../../store/pdfFiles/selectors';
+import connect from 'react-redux/es/connect/connect';
+import { Document, Page, pdfjs } from 'react-pdf';
+
+import './Inspect.scss';
+
+pdfjs.GlobalWorkerOptions.workerSrc = '../pdf.worker.js';
```

Comment:
Probably using relative path here is not quite safe in case we decide to make route deeper. We can use the following variable instead:
```
const { PUBLIC_URL } = process.env;
pdfjs.GlobalWorkerOptions.workerSrc = `${PUBLIC_URL}/pdf.worker.js`;
```